internal/cmd/templates.go

```diff
@@ -45,6 +45,7 @@ var (
 			return runner.All(args, flags.disableFormat, flags.disableLint, flags.fix)
 		},
 		BindFlags: func(flagSet *pflag.FlagSet, flags *flags) {
+			flags.bindConfigData(flagSet)
```

Comment:
Why not just make this a global flag, similar to `debug`? I realize that some commands might not need a configuration, such as `version` or `config init`, but the same could be said about `debug`, so adding this to the global set seems fine to me. What are your thoughts?

Reply:
This isn't global, by the same argument, `--protoc-url` should be global and we chose that not to be global.